code/modules/mob/living/carbon/human/dummy.dm

```diff
@@ -57,20 +57,23 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy)
 			to_nuke += checking
 			continue
 
+		if(istype(checking, /obj/item/pda))
+			var/obj/item/pda/whats_shitcode = checking
```

Comment:
ahem

Reply:
damn this isnt even yours this sucks

Reply:
(he doesn't know)

Reply:
I should put this on move logic, if you remove a pen from the pda it should lose the ref no matter what

Reply:
its just a bad name

Reply:
no like outside of the name this is shitty code

Reply:
Shit is unfuck